UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.Common/HealthCheck/AzureWebJobHelper.cs

```diff
@@ -0,0 +1,61 @@
+﻿
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Newtonsoft.Json;
+
+namespace UKHO.ExternalNotificationService.Common.HealthCheck
+{
+    [ExcludeFromCodeCoverage]
+    public class AzureWebJobHelper : IAzureWebJobHelper
+    {
+        static HttpClient httpClient = new HttpClient();
+        private readonly IWebHostEnvironment _webHostEnvironment;
+
+        public AzureWebJobHelper(IWebHostEnvironment webHostEnvironment)
+        {
+            _webHostEnvironment = webHostEnvironment;
+        }
+
+        public async Task<HealthCheckResult> CheckWebJobsHealth(WebJobDetails webJob)
+        {
+            try
+            {
+                string webJobDetail, webJobStatus = string.Empty;
+
+                using HttpRequestMessage httpRequestMessage = new HttpRequestMessage(HttpMethod.Get, webJob.WebJobUri);
```

Comment:
new assignment can be simplified as new(HttpMethod.Get, webJob.WebJobUri);

Reply:
Changes done